lib/matplotlib/tests/test_usetex.py

```diff
@@ -131,7 +131,7 @@ def test_missing_psfont(fmt, monkeypatch):
     monkeypatch.setattr(
         dviread.PsfontsMap, '__getitem__',
         lambda self, k: dviread.PsFont(
-            texname='texfont', psname='Some Font',
+            texname=b'texfont', psname=b'Some Font',
```

Comment:
I'm slightly worried about this change is it implies we had some str -> bytes change somewhere else?